packages/api/lambdas/sf-starter.js

```diff
@@ -178,6 +178,7 @@ async function sqs2sfEventSourceHandler(event) {
 }
```

Comment:
This is the handler method for `sqs2sf` lambda, and Event Source Mapping is used.  It doesn't call `consumer.processMessage`.
We can update `handleSourceMappingEvent` to check the error.  Also we should consider return batchItemFailures and add to the event source mapping terraform configuration:
function_response_types = ["ReportBatchItemFailures"]

Reply:
Yeah taking a closer look, I don't think that handler method is being used anywhere... 🤔 I think I should remove it. 

I can update the event source mapping like you suggested. https://github.com/nasa/cumulus/pull/3389/commits/20827cb57cebbc9ed1706e951e4a2785a5cbf066

Reply:
When we use `ReportBatchItemFailures` response type, we need to update the lambda to return `batchItemFailures`, so that if only failed messages will remain in the queue and get retried, rather than fail the whole batch.
https://github.com/nasa/cumulus/blob/master/packages/api/lambdas/sf-event-sqs-to-db-records/index.js#L142